pkgs/tools/text/ispell/default.nix

```diff
@@ -22,8 +22,8 @@ stdenv.mkDerivation rec {
     ./patches/0030-Display-whole-multibyte-character.patch
   ];
   postPatch = ''
+    if [ ! stdenv.isDarwin ]; then echo "#define USG" >> local.h; fi
```

Comment:
that conditional isn't available in bash, you want something like this

```nix
{
  postPatch = ''
    cat >> local.h <<EOF
    ${stdenv.lib.optionalString (!stdenv.isDarwin) "#define USG"}
    #define TERMLIB "-lncurses"
    ...
    EOF
  '';
}
```

Reply:
Thanks for the suggestion.